model/src/test/kotlin/OrtResultTest.kt

```diff
@@ -307,7 +307,7 @@ class OrtResultTest : WordSpec({
                         )
                     )
                 ),
-                evaluator = EvaluatorRun(
+                evaluator = EvaluatorRun.EMPTY.copy(
```

Comment:
Nit: The argument "for specifying only the needed arguments" is not really convincing here, as no unneeded arguments can be dropped here.